enhancements/aws-pod-identity.md

```diff
@@ -0,0 +1,442 @@
+---
+title: aws-pod-identity 
+authors:
+  - "@sjenning"
+reviewers:
```

Comment:
why no reviewers from the OWNERS of the cloud-cred-operator?

Reply:
i'll add some